frontend/benefit/applicant/src/components/applications/pageContent/PageContent.tsx

```diff
@@ -46,6 +59,55 @@ const PageContent: React.FC = () => {
     );
   }
 
+  // if view mode, show customized summary
+  if (
+    application.status &&
+    SUBMITTED_STATUSES.includes(application.status) &&
+    Boolean(isReadOnly)
+  ) {
+    return (
+      <Container>
+        <$PageHeader>
+          <$HeaderItem>
+            <$PageHeading>
+              {t('common:applications.pageHeaders.edit')}
+            </$PageHeading>
+          </$HeaderItem>
+        </$PageHeader>
+        {id && application?.submittedAt && application?.applicationNumber && (
+          <$PageSubHeading
+            css={`
+              font-weight: 400;
+              font-size: ${theme.fontSize.body.m};
+            `}
+          >
+            {t('common:applications.pageHeaders.sent', {
+              applicationNumber: application.applicationNumber,
+              submittedAt: formatDate(
```

Comment:
date.utils has `convertToUIDateAndTimeFormat(application?.submittedAt)` method which might be easier

Reply:
There is quite much repetition over editable and non-editable, prefereably these components could be combined somehow, now it's getting hard to read this.

Reply:
there are multiple scenarios: new, edit, view, error from axios, error from view mode.. i was thinking first to create a separate page but managed to wrap whole functionality in one.

Reply:
fixed